src/components/Activity/Action.tsx

```diff
@@ -87,7 +87,7 @@ const ActionOffer: FunctionComponent<Props> = ({ action, verbose }) => (
 const ActionOfferAccepted: FunctionComponent<Props> = ({ action }) => (
   <>
     <UserBadge className={cs(style.user)} hasLink={false} user={(action.issuer||action.target)!} size="regular" />
-    <span>🔄 traded its <strong>token {getTokenIdx(action.objkt?.name!)}</strong></span>
```

Comment:
This action refers to to a user who sold its token.
This change implies that the one who sold it purchased it